views/room.ejs

```diff
@@ -319,7 +319,7 @@
           <div class="google-sign-in-note-container">
             <div class="google-sign-in-note">
               <span class=title>Note:</span><br/>
-              <span>This function requires you to sign in with your <b>@tokbox.com</b> Google account</span>
+              <span>This function requires you to sign in with your <b>@tokbox.com</b> or <b>@vonage.com</b> Google account</span>
```

Comment:
Please generate this string programmatically, based on the domain name(s). Do not hard-code in `@tokbox.com` and `@vonage.com`.